From 874453845058ae2061769e4cb2c0ec86835730db Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Thu, 9 Dec 2010 00:55:35 +0800
Subject: [PATCH 13/59] ENGR00136091 vpu: Change Android makefile for "tr"
 command usage.

Change makefile to make it compatible with old "tr" version.

Signed-off-by: Sammy He <r62914@freescale.com>
---
 firmware/vpu/Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/firmware/vpu/Android.mk b/firmware/vpu/Android.mk
index 592d69d..e0c86b7 100644
--- a/firmware/vpu/Android.mk
+++ b/firmware/vpu/Android.mk
@@ -6,7 +6,7 @@ LOCAL_PATH := $(call my-dir)
 
 vpu_fw_target := $(TARGET_OUT)/lib/firmware/vpu
 
-soc := $(shell echo "$(BOARD_SOC_TYPE)" | tr [A-Z] [a-z])
+soc := $(shell echo "$(BOARD_SOC_TYPE)" | tr 'A-Z' 'a-z')
 vpu_fw_file := vpu_fw_$(soc).bin
 
 # Firmware
-- 
1.8.0

